### ISSUE TITLE
ss/ß should be treated the same in search

### ISSUE THREAD

**Issue by martinantonmueller** [acdh-oeaw/apis-core#186]
Currently: ss and ß are not the same for the PMB-search. So searching for "Ress" won't find "Reß" and the other way round. ß and ss should be interchangeable

**Comment by skurzinz** [comment]
Obviously this generates questions as to the handling of Umlauts (ae = ä etc.) and of diacritics in other languages (sz = ß/s in German, but Hungarian sz = German s and Polish sz = Croatian š = German sch). It's likely we're not the first ones to think about this, but nevertheless should think wisely :)

**Comment by martinantonmueller** [comment]
you're right! maybe this can be treated as language specific? In PMB it would make sense to have the German handlings.
